csrc/device_lower/lower2device.h

```diff
@@ -214,6 +214,14 @@ class GpuLower : public NonCopyable {
     return requires_id_model_;
   }
 
+  bool hasMmaOps() const {
+    return has_mma_ops_;
+  }
```

Comment:
I don't like this logic: "if matmul kernel, then do this special thing".

Reply:
Without this special thining matmul tests fails e.g. `MatmulTest.AmpereMatmulTNCpAsync\n` in [CI](https://nv/e2E/113447711). I believe the validation error is becuase: for matmul we need to use inline predicate in cpAsync to ensure out of boundary positions are filled with 0, these out of boundy positions are usd in mma. Are there other options to avoid this err?

Reply:
Then why matmuls are special? How can we assume that other kernels do not need them to be filled with zero?